README.md

```diff
@@ -0,0 +1,17 @@
+
+
+## Installing
+
+To install all dependent packages, use the below command
+
+```
+npm install
+```
+
+## Building
+
+To compile the source files, use the below command
+
+```
+npm run build
+```
```

Comment:
Add running comment, refer the below ReadMe for further details.

https://github.com/syncfusion/ej2-samples/#running